os/kernel/debug/mem_leak_checker.c

```diff
@@ -288,9 +294,15 @@ static void ram_check(char *bin_name, int *leak_cnt, uint32_t *bin_text_addr)
 	
 	search_addr(&_sdata, &_edata, leak_cnt);
 	search_addr(&_sbss, &_ebss, leak_cnt);
+
+#ifdef CONFIG_ARCH_BOARD_RTL8720E
```

Comment:
I don't agree adding board specific in the common code. Let's find other way.

Reply:
Have added a script file, that generates a .c file during build phase, if the mem leak config is enabled, which has array, which is used for searching.